CenterForOpenScience/ember-osf-web
# [Feature][ENG-94] Testimonial carousel

## Purpose

To add a testimonial carousel for the logged-out homepage

## Summary of Changes

- Add `testimonial` section on logged-out homepage
- Create `carousel` component

<img width="1262" alt="Screen Shot 2019-06-12 at 1 20 45 PM" src="https://user-images.githubusercontent.com/19379783/59372220-ff15ec80-8d14-11e9-88fd-95abac95ebb7.png">

<img width="499" alt="Screen Shot 2019-06-12 at 1 21 14 PM" src="https://user-images.githubusercontent.com/19379783/59372225-02a97380-8d15-11e9-8523-5c79b5ed0195.png">


## Side Effects

`N/A`

## Feature Flags

`N/A`

## QA Notes

This will be a part of the logged-out homepage.  To get there, you'll have to use the `zoom-to-route` function in the developer footer and go to `new-home`.  There won't be any special functionality with the carousel, but please be sure to check that the `See his/her research` links go to the right places and that the carousel works as expected.  It should also be responsive on all devices and should scale down appropriately.

## Ticket

https://openscience.atlassian.net/browse/ENG-94

# Reviewer Checklist
